src/main/java/ladder/view/ResultView.java

```diff
@@ -2,96 +2,95 @@
 
 import ladder.game.LadderGame;
 import ladder.game.Participants;
-import ladder.game.Prizes;
+import ladder.game.Prize;
 import ladder.structure.Ladder;
 import ladder.structure.LineOfLadder;
+import ladder.structure.connection.result.Point;
 
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
 public class ResultView {
     private static final String LADDER_RESULT = "사다리 결과";
     private static final String GAME_RESULT = "실행결과";
     private static final String EMPTY = "";
-    private static final String SINGLE_SPACE = " ";
-    private static final String DOUBLE_SPACE = "  ";
+    private static final String PADDING_LADDER = "    ";
     private static final String VERTICAL = "|";
     private static final String HORIZON = "-----";
     private static final String EMPTY_HORIZON = "     ";
-    private static final String PARTICIPANT_FORMAT = "%s%s%s";
-    private static final String RESULT_FORMAT = "%s%s%s";
+    private static final String PARTICIPANT_FORMAT = "%6s";
+    private static final String RESULT_FORMAT = "%6s";
     private static final String GAME_RESULT_FORMAT = "%s : %s";
-    private static final int NAME_SPACE_SIZE = 6;
+    private static final String ALL = "all";
 
     public static void showLadderResult(LadderGame ladderGame) {
         System.out.println(LADDER_RESULT);
         showParticipant(ladderGame.getParticipants());
         drawLadder(ladderGame.getLadder());
-        showResults(ladderGame.getPrizes());
+        showResults(ladderGame.getPrizes().values());
     }
 
-    public static void showResultOfParticipant(LadderGame ladderGame, String name) {
+    public static boolean showResultOfParticipant(LadderGame ladderGame, String name) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    public static boolean showResultOfParticipant(Results results, String name) {
```